File: Programming Basics - Python/Conditional Statements Advanced - Lab/05.SmallShop.py
product = input()
city = input()
quantity = float(input())

product_price = 0

if product == 'coffee':
    if city == 'Sofia':
        product_price = 0.50
    elif city == 'Plovdiv':
        product_price = 0.40
    elif city == 'Varna':
        product_price = 0.45

elif product == 'water':
    if city == 'Sofia':
        product_price = 0.80
    elif city == 'Plovdiv' or city == 'Varna':
        product_price = 0.70

elif product == 'beer':
    if city == 'Sofia':
        product_price = 1.20
    elif city == 'Plovdiv':
        product_price = 1.15
    elif city == 'Varna':
        product_price = 1.10

elif product == 'sweets':
    if city == 'Sofia':
        product_price = 1.45
    elif city == 'Plovdiv':
        product_price = 1.30
    elif city == 'Varna':
        product_price = 1.35

elif product == 'peanuts':
    if city == 'Sofia':
        product_price = 1.60
    elif city == 'Plovdiv':
        product_price = 1.50
    elif city == 'Varna':
        product_price = 1.55

print(product_price * quantity)

File: Programming Basics/Conditional Statements Advanced - Lab/05.SmallShop.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        string product = Console.ReadLine();
        string city = Console.ReadLine();
        double quantity = double.Parse(Console.ReadLine());

        goods(product, city, quantity);
    }

    static void goods(string product, string city, double quantity)
    {
        double productPrice = 0;

        switch (city)
        {
            case "Sofia":
                switch (product)
                {
                    case "coffee":
                        productPrice = 0.50;
                        break;
                    case "water":
                        productPrice = 0.80;
                        break;
                    case "beer":
                        productPrice = 1.2;
                        break;
                    case "sweets":
                        productPrice = 1.45;
                        break;
                    case "peanuts":
                        productPrice = 1.60;
                        break;
                }
                break;
            case "Plovdiv":
                switch (product)
                {
                    case "coffee":
                        productPrice = 0.40;
                        break;
                    case "water":
                        productPrice = 0.70;
                        break;
                    case "beer":
                        productPrice = 1.15;
                        break;
                    case "sweets":
                        productPrice = 1.30;
                        break;
                    case "peanuts":
                        productPrice = 1.50;
                        break;
                }
                break;
            case "Varna":
                switch (product)
                {
                    case "coffee":
                        productPrice = 0.45;
                        break;
                    case "water":
                        productPrice = 0.70;
                        break;
                    case "beer":
                        productPrice = 1.10;
                        break;
                    case "sweets":
                        productPrice = 1.35;
                        break;
                    case "peanuts":
                        productPrice = 1.55;
                        break;
                }
                break;
        }
        Console.WriteLine(productPrice * quantity);
    }
}